Packs/FireMonSecurityManager/IncidentFields/incidentfield-FireMonSecurityManager-Action.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
@tallieber Correct if I wrong but I don't think that is the correct form for JSON field.

Reply:
right, I will handle it in the internal PR